### PR TITLE
Add bool type support, Fixes #5

### LINKED ISSUE #5: bool type not supported
`bool` type is not recognized and throws the following exception:

    Exception: Found wire type 0 (varint), wanted type 2 (bool)

### PR DESCRIPTION
**Pull request by JordanGoulder**

### CODE REVIEW

File: lib/types.py [hunk 1]
```diff
@@ -181,6 +181,10 @@ def parse_uint64(self, x, type):
         assert(0 <= x < (1 << 64))
         return fg3(str(x))
 
+    def parse_bool(self, x, type):
+        assert(0 <= x < (1 << 2))
```
Comment (mildsunrise) [inline lib/types.py]: Thanks for the fix! I think this should be `0 <= x < (1 << 1))`?

Comment (JordanGoulder) [reply lib/types.py]: Ooops! Yeah, that is correct!

Comment (JordanGoulder) [reply lib/types.py]: Would you like me to close the pull request and submit a new one?

**Comment by mildsunrise** [comment]
Merged, thanks :)

**Pull request merged**